### PR TITLE
Courses tests

### PR DESCRIPTION
**Pull request by Reactor94**
- added Readme
- added pages for Media, Courses, Words, Users, and common where we have reused components
- added navigation actions and for courses
- added 2 tests for courses

### CODE REVIEW

File: cypress/integration/course.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+describe('Courses tests', () => {
+
+    beforeEach(() => {
+        const email = Cypress.env('email');
+        const password = Cypress.env('password');
+        login(email, password);
+   });
+
+    it('Should create new course and check if it exists', () => {
+        const courseSlug = generateName('course');
```
Comment (wojciechnowacki) [inline cypress/integration/course.spec.js:17]: use faker instead hardcoding stuff ;)

File: cypress/integration/course.spec.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        navigateToCourses();
+        createNewCourse(courseSlug, 1);
+        assertCourseExists(courseSlug, 1 );
+    });
+
+    it('Should check if admin can edit created course', () => {
+        const courseSlug = generateName('slug');
+        const editedSlug = generateName('edited');
```
Comment (wojciechnowacki) [inline cypress/integration/course.spec.js:25]: same here

File: cypress/integration/course.spec.js (hunk 1 above)
Comment (Reactor94) [reply cypress/integration/course.spec.js]: fixed

File: cypress/integration/course.spec.js (hunk 2 above)
Comment (Reactor94) [reply cypress/integration/course.spec.js]: fixed

**Pull request merged**